br/pkg/gluetidb/glue.go

```diff
@@ -158,6 +158,10 @@ func (g Glue) UseOneShotSession(store kv.Storage, closeDomain bool, fn func(glue
 	return nil
 }
 
+func (_ Glue) GetClient() int {
+	return glue.BrClient
```

Comment:
Prefer `ClientBR` and `ClientSQL` so when we are typing `Client` the auto complete will hint us what all the client types are. 
(Also in `go`, we prefer make first letter abbreviates all upper case, say, `io.EOF`)